### PR TITLE
Bump nightlies to 1.8.0

### PR DESCRIPTION
**Pull request by seemethere**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#45696 Bump nightlies to 1.8.0**

Similar to https://github.com/pytorch/pytorch/pull/40519

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

Differential Revision: [D24064381](https://our.internmc.facebook.com/intern/diff/D24064381)

### CODE REVIEW

**Comment by xuzhao9** [comment]
In this PR, do we also need to bump the version numbers hardcoded in all the files modified in [#40519](https://github.com/pytorch/pytorch/pull/40519/files), e.g., torch/csrc/onnx/onnx.h?

**Comment by seemethere** [comment]
> In this PR, do we also need to bump the version numbers hardcoded in all the files modified in [#40519](https://github.com/pytorch/pytorch/pull/40519/files), e.g., torch/csrc/onnx/onnx.h?

You're absolutely right, updating

File: android/test_app/app/build.gradle [hunk 1]
```diff
@@ -149,8 +149,8 @@ dependencies {
     //nativeBuildImplementation(name: 'pytorch_android_torchvision-release', ext: 'aar')
     //extractForNativeBuild(name: 'pytorch_android-release', ext: 'aar')
 
-    nightlyImplementation 'org.pytorch:pytorch_android:1.7.0-SNAPSHOT'
```
Comment (walterddr) [inline android/test_app/app/build.gradle:152]: do we need to bump version in `android/gradle.properties` ?

Comment (seemethere) [reply android/test_app/app/build.gradle:152]: Good catch on that one as well, updating

**Review by samestep — APPROVED** [review]

**Review by malfet — APPROVED** [review]

**Pull request closed without merging**